README.md

```diff
@@ -49,7 +49,17 @@ V = 10
 V = 6
 |E| = 12
 ```
-> Note: You need to convert SC2 sentence into FO2 sentence with cardinality constraints by yourself.
+
+> Note: You can also directly input the SC2 sentence
+
+2 regular graphs (sc2):
+```
+\forall X: (~E(X,X)) &
+\forall X: (\forall Y: (E(X,Y) -> E(Y,X))) &
+\forall X: (\exists_{=2} Y: (E(X,Y)))
+
+V = 6
+```
```

Comment:
I realized we were overcounting the models for SC2 (see the equation below). We should handle it before outputting the result.
<img width="882" alt="image" src="https://github.com/lucienwang1009/lifted_sampling_fo2/assets/28888626/5c406754-7ea1-4533-a4eb-0dabec1e9ec1">